win/rl/pynethack.cc

```diff
@@ -177,14 +177,25 @@ class Nethack
         }
     }
 
+    void
+    set_spawn_monsters(bool spawn_monsters)
+    {
+#ifdef NLE_ALLOW_CONTROL
+        init_settings_.spawn_monsters = spawn_monsters;
+        use_init_settings = true;
+#else
+        throw std::runtime_error("Setting monster spawning not enabled");
+#endif
+    }
+
     void
     set_initial_seeds(unsigned long core, unsigned long disp, bool reseed)
     {
-#ifdef NLE_ALLOW_SEEDING
-        seed_init_.seeds[0] = core;
-        seed_init_.seeds[1] = disp;
-        seed_init_.reseed = reseed;
-        use_seed_init = true;
+#ifdef NLE_ALLOW_CONTROL
+        init_settings_.seeds[0] = core;
+        init_settings_.seeds[1] = disp;
+        init_settings_.reseed = reseed;
+        use_init_settings = true;
```

Comment:
Ah, this is a bit dangerous I think. `use_seed_init` used to change the seed arguments form the default. We might have to change this a bit.